dashboard/app/views/devise/shared/_links.haml

```diff
@@ -1,29 +1,46 @@
--# Unless we're already on the Sign in page, show a Sign in link
-- unless ['sessions', 'registrations'].include? controller_name
-  = t('auth.already_signedup')
-  = link_to t('nav.user.signin'), new_session_path(resource_name)
-  %br/
-  %br/
+.devise_link_section
+  .or_break
+    .vertical_bar
+    %p
+      OR
+    .vertical_bar
 
-- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks'
-  = link_to t('auth.need_unlock'), new_unlock_path(resource_name)
-  %br/
-  %br/
+  %div{style: "display: block;"}
```

Comment:
The diff got mangled here, what's really happening is everything that was previously here is put inside an enclosing div without edits (everything down to line 71 is essentially untouched apart from indentation).